tests/scaffoldings/dotd/custom/extras/goodconf.py

```diff
@@ -0,0 +1,4 @@
+#!/usr/local/bin/python-odoo-shell
+from odoo.tools import config
+assert config.get("test_config_setting") == "Over9000"
```

Comment:
[It says line 3 fails](https://travis-ci.org/Tecnativa/docker-odoo-base/jobs/327450411#L4695-L4696), so this is possibly not doing what you expect.

Reply:
I'm actually not sure why I'm even creating that file or testing this part. Totally unrelated to the task at hand. Fixed.

If tests pass though, I think this might mean there's an issue with config generation in the custom directories.